src/v/kafka/protocol/schemata/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ set(schemata
   describe_acls_response.json
   describe_log_dirs_request.json
   describe_log_dirs_response.json
+  describe_producers_request.json
```

Comment:
add to commit messages from which kafka version you copied the schema file. this will make it easier to orient ourselves later if needed.